web/frontend/vue.config.js

```diff
@@ -23,6 +23,7 @@ let vueConfig = {
     resolve: {
       alias: {
         '@src': path.join(__dirname, 'src'),
+        '@config': path.join(__dirname, 'src/config'),
```

Comment:
I remember you just cleaned up the aliases and consolidated them to just `@src`. Why don't we just keep on using `@src`?

Reply:
It's config file, this is the only place we can't use aliases, because we define them here